content/en/docs/Installation/_index.md

```diff
@@ -434,13 +434,13 @@ The following flags are used to control the behavior of Kyverno and must be set
 
 1. `excludeGroupRole`: excludeGroupRole role expected string with comma-separated group role. It will exclude all the group role from the user request. Default we are using `system:serviceaccounts:kube-system,system:nodes,system:kube-scheduler`.
 2. `excludeUsername`: excludeUsername expected string with comma-separated kubernetes username. In generate request if user enable `Synchronize` in generate policy then only kyverno can update/delete generated resource but admin can exclude specific username who have access of delete/update generated resource.
-3. `resourceFilters`: Kubernetes resources in the format "[kind,namespace,name]" where the policy is not evaluated by the admission webhook. For example --filterKind "[Deployment, kyverno, kyverno]" --filterKind "[Deployment, kyverno, kyverno],[Events, *, *]".
-4. `generateSuccessEvents`: specifies whether (true/false) to generate success events. Default is set to "false".
+3. `generateSuccessEvents`: specifies whether (true/false) to generate success events. Default is set to "false".
```

Comment:
Do we need to explicitly number the list elements ?
Markdown supports `1.` for all elements in the list, it will show the correct numbers when rendering.

Reply:
It doesn't matter either way. VS Code just auto increments numbered lists. They all come out the same way.

Reply:
Ok, I found it easier to always `1.`, makes moving elements around easier.